alchemist-incarnation-protelis/src/main/java/it/unibo/alchemist/model/ProtelisIncarnation.java

```diff
@@ -214,7 +222,24 @@ public Molecule createMolecule(final String s) {
             final Environment<Object, P> environment,
             final String parameter
     ) {
-        return new ProtelisNode<>(environment);
+        final Node<Object> node = new GenericNode<>(this, environment) {
+
+            @Override
+            protected Object createT() {
+                return createConcentration();
+            }
+
+            @NotNull
+            @Override
+            public Node<Object> cloneNode(@NotNull final Time currentTime) {
+                final Node<Object> result = createNode(randomGenerator, environment, parameter);
```

Comment:
does this call also populate the properties?

`NodeProperties` must be cloneable with the `cloneOnNewNode` mechanism